packages/context/src/lib/decorators/provide.ts

```diff
@@ -16,14 +16,10 @@ import {ContextProvider} from '../controllers/context-provider.js';
  */
 
 /**
- * A property decorator that adds a ContextConsumer controller to the component
- * which will try and retrieve a value for the property via the Context API.
+ * A property decorator that adds a ContextProvider controller to the component
+ * making it respond to any `context-request` events from its children consumer.
  *
  * @param context A Context identifier value created via `createContext`
- * @param multiple An optional boolean which when true allows the value to be updated
- *   multiple times.
- *
- * See: https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector
  *
  * @example
  *
```

Comment:
Nit: in the example directly below, change `consume` to `provide` (in the imports, L27).

Reply:
Thank you for catching that!

Reply:
I also want to change usage example below this.
```js
 * class MyElement {
 *   @provide({context: loggerContext})
 *   logger;
 *
 *   doThing() {
 *     this.logger.log('thing was done');
 *   }
 * }
```
`doThing()` is more what a consumer would do. Provider should set an initial value on the decorated property.. but going with the theme here would that be something like this?
```js
 * class MyElement {
 *   @provide({context: loggerContext})
 *   logger = new DefaultLogger();
 * }
 ```

Also wondering if the logger dependency injection theme here is better than just some data object.

Reply:
Yes. I much prefer this because it's also more similar to the example we give in the lit.dev documentation.

Reply:
Thanks! Updated to match the example on lit.dev docs.